### PR TITLE
kokoro: avoid git permissions issue that GN fails on

### PR DESCRIPTION
**Pull request by dneto0**
In the docker build script that Kokoro runs, the directories are owned by a different user.  Git complains about that and in the GN flow GN will error out.
In this docker flow we don't care about that warning, so within the docker config set a git global option to ignore the issue.

### CODE REVIEW

**Review by dj2 — APPROVED** [review]

**Comment by jeremy-lunarg** [comment]
@dneto0 Should we expect the CI-ubuntu-clang-gn job to pass in this PR? Or will that job start passing once this PR lands on main?

**Comment by dneto0** [comment]
> @dneto0 Should we expect the CI-ubuntu-clang-gn job to pass in this PR? Or will that job start passing once this PR lands on main?

Sigh. Now it's failing for a different reason.

The ninja build tool was removed from depot_tools.  I'll patch this PR.

**Comment by dneto0** [comment]
> @dneto0 Should we expect the CI-ubuntu-clang-gn job to pass in this PR? Or will that job start passing once this PR lands on main?

The intent is to land only after that bot works in this PR.
thanks

File: kokoro/linux-clang-gn/build-docker.sh [hunk 1]
```diff
@@ -36,6 +36,12 @@
 set -e # Fail on any error.
 set -x # Display commands being run.
 
+using ninja-1.10.0
```
Comment (dneto0) [inline kokoro/linux-clang-gn/build-docker.sh:39]: well that didn't work.

Comment (jeremy-lunarg) [reply kokoro/linux-clang-gn/build-docker.sh:42]: The job is currently passing. Is there something else that is broken?

Comment (jeremy-lunarg) [reply kokoro/linux-clang-gn/build-docker.sh:42]: Ah, I see. I believe you were referring to the missing "using" declaration. Everything appears to be okay now. Can I proceed with the merge?

**Review by jeremy-lunarg — APPROVED** [review]
LGTM

**Pull request merged**